pocs/camera/camera.py

```diff
@@ -214,8 +213,10 @@ def take_observation(self, observation, headers=None, filename=None, *args, **kw
                 observation.exposure_list[image_id] = file_path
 
         # Process the exposure once readout is complete
-        t = Thread(target=self.process_exposure, args=(
-            metadata, observation_event, exposure_event))
+        t = threading.Thread(
+            target=self.process_exposure,
+            args=(metadata, observation_event, exposure_event),
+            daemon=True)
```

Comment:
I think for the camera and some of these others I was following the advice given in the note [here](https://docs.python.org/3/library/threading.html#thread-objects). I can't think of any unintended consequences of daemonizing this with the camera but seems like it would be hard to test until we have one fail.